### ISSUE TITLE
fatal error: nonstd/expected.hpp: No such file or directory

### ISSUE THREAD

**Issue by KonradHoeffner** [TortugaAttack/rdf4cpp#210]
**Describe the bug**
Compiling a program using rdf4cpp results in:

```
$gcc test.cpp
In file included from /usr/local/include/rdf4cpp/rdf/datatypes/registry/LiteralDatatypeImpl.hpp:4,
                 from /usr/local/include/rdf4cpp/rdf/datatypes/xsd/Base64Binary.hpp:5,
                 from /usr/local/include/rdf4cpp/rdf/datatypes/xsd.hpp:4,
                 from /usr/local/include/rdf4cpp/rdf/storage/node/identifier/NodeID.hpp:8,
                 from /usr/local/include/rdf4cpp/rdf/storage/node/identifier/NodeBackendHandle.hpp:4,
                 from /usr/local/include/rdf4cpp/rdf/storage/node/NodeStorage.hpp:4,
                 from /usr/local/include/rdf4cpp/rdf/Node.hpp:9,
                 from /usr/local/include/rdf4cpp/rdf/IRI.hpp:5,
                 from /usr/local/include/rdf4cpp/rdf/Namespace.hpp:4,
                 from /usr/local/include/rdf4cpp/rdf/ClosedNamespace.hpp:4,
                 from /usr/local/include/rdf4cpp/rdf.hpp:4,
                 from test.cpp:1:
/usr/local/include/rdf4cpp/rdf/datatypes/LiteralDatatype.hpp:11:10: fatal error: nonstd/expected.hpp: No such file or directory
   11 | #include <nonstd/expected.hpp>
```

**To Reproduce**
1. Install rdf4cpp to the system as described in the README.md.
2. `echo "#include <rdf4cpp/rdf.hpp>" > test.cpp`
3. `gcc test.cpp`

**Expected behavior**
It should compile.

**Desktop (please complete the following information):**
 - OS: Arch Linux
 - gcc 13.2.1

P.S.: I'm not experienced in C++ so this may have some easy fix I don't know about.

**Comment by bigerl** [comment]
That is rather an issue with the Readme. The problem is that rdf4cpp uses headers from other libraries in its public interface. You would need to include them as well. I would suggest to use conan instead to generate an integration with the build system of your choice. Check conan generators to see what it supports.

**Comment by bigerl** [comment]
TODO: clarify in the readme

**Comment by liss-h** [comment]
Resolved by #206
